src/posit/connect/content.py

```diff
@@ -17,6 +17,10 @@
 class ContentItem(Resource):
     """A piece of content."""
 
+    @property
+    def id(self) -> str:
```

Comment:
Only looking at this since you moved it: should we actually remove this accessor? Is my understanding correct that `id` is effectively deprecated and that `guid` is the real identifier you should always use? @aronatkins?

Reply:
We originally did not include `id` in the public v1 API. The `id` was added afterwards in order to help with cross-referencing against the on-disk layout and server logs (which did not include guid at that time).

At this point, it is probably too strong a statement to declare `id` as deprecated, but it cannot be used to look-up content items using the v1 API.